docs/plugins.md

```diff
@@ -50,7 +50,7 @@ Any transforms in stage-x presets are changes to the language that haven't been
   </p>
 </blockquote>
 
-The [TC39](https://github.com/tc39) categorises proposals into 4 stages:
```

Comment:
Maybe a nit, but are we replacing the British spelling for words? I personally don't have a preference, only that we be consistent